Engine/Module/ModuleLight.cpp

```diff
@@ -41,7 +41,7 @@ void ModuleLight::RenderDirectionalLight(const float3& mesh_position)
 {
 	SortClosestLights(mesh_position, ComponentLight::LightType::DIRECTIONAL_LIGHT);
 	current_number_directional_lights_rendered = 0;
-	int i = 0;
+	unsigned int i = 0;
```

Comment:
you may can use size_t better. Is the same but shorter

Reply:
Thats right